src/cljs/replumb/repl.cljs

```diff
@@ -570,6 +570,25 @@
       (call-back (common/wrap-success (s/join \newline vars)))
       (call-back (common/wrap-success "nil")))))
 
+(defn process-apropos
+  "Given a regular expression or stringable thing, return a seq of all
+  public definitions in all currently-loaded namespaces that match the
+  str-or-pattern."
+  [opts cb data str-or-pattern]
+  (let [matches? (if (instance? js/RegExp str-or-pattern)
```

Comment:
`apropos` can take either a string or a regex. I ported form [here](https://github.com/clojure/clojurescript/blob/c72e9c52156b3b348aa66857830c2ed1f0179e8c/src/main/clojure/cljs/repl.cljc#L1228). Maybe double check if the JS version is correct, should be.